.github/actions/build-to/action.yml

```diff
@@ -0,0 +1,33 @@
+name: 'Build workspaces'
```

Comment:
`action.yml` seems a bit too generic of a name

Reply:
action action action ? ofc 🔢 :D